.github/workflows/ci-dgraph-code-coverage.yml

```diff
@@ -22,82 +22,8 @@ on:
     branches:
       - main
       - 'release/**'
-  pull_request_target:
-    types:
-      - opened
-      - reopened
-      - synchronize
-      - ready_for_review
-    branches:
-      - main
-      - 'release/**'
-  schedule:
```

Comment:
What's the reason to remove the scheduled workflow execution?

Reply:
Code coverage results only change when the code changes (i.e. when code is pushed to main).  Running on a schedule would just send the same code coverage results repeatedly and waste CPU resources (since this is run on our self-hosted runners).

Not sure why this was already here, but presumably we copied it from another workflow and overlooked it.

Reply:
We could also change it so that is runs once a day on a schedule (instead of 4 times a day, which seems wasteful).

Reply:
Makes sense to only run when the code changes.

Reply:
running once for a PR when someone has reviewed is sufficient IMO